### ISSUE TITLE
Should we remove legacy code that no understands?

### ISSUE THREAD

**Issue by jjmaldonis** [uw-cmg/StructOpt#6]
Right now I think there are some code blocks that no one understands. They are all in version control now, so should we just delete them? This would help us keep track of the working code more easily imo.

**Comment by jjmaldonis** [comment]
For example, there are a whole bunch of `fuss` files in `selection`. Do we need/want them all?

There are also files with the names: `fitpred.py`, `fitpred_bests.py`, `fitpred_new.py` in the `predator` directory. Do we want all three of those? Or is `fitpred_new.py` the "new" one that we want?

Same with `io/read_xyz_v0.py` and `io/read_xyz.py`. Can we avoid this since they are all in version control anyways?

**Comment by ZhewenSong** [comment]
I agree removing older versions of code like `io/read_xyz_v0.py`. But something like `fuss` is just a selection scheme and I don't think we should remove them at all.
